azure/template.json

```diff
@@ -77,10 +84,45 @@
                     },
                     "storageKind": {
                         "value": "[parameters('storageKind')]"
+                    },
+                    "enableCors": {
+                        "value": "[parameters('enableCors')]"
                     }
                 }
             },
-            "dependsOn": []
+            "dependsOn": [
+            ]
+        },
+        {
+            "apiVersion": "2017-05-10",
+            "name": "[concat('storage-container', parameters('containerArray')[copyIndex()].name)]",
+            "type": "Microsoft.Resources/deployments",
+            "condition": "[greater(length(parameters('containerArray')), 0)]",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/master/templates/storage-container.json",
```

Comment:
Could a deploymentUrlBase variable be used?

Reply:
Possibly this is a a task to replace a complex poweshell script. Not a complete re-implementation of the service